spec/lib/saml/saml_settings_service_spec.rb

```diff
@@ -1,36 +1,51 @@
 # frozen_string_literal: true
 require 'rails_helper'
 require 'saml/settings_service'
+require 'lib/sentry_logging_spec_helper'
 
 RSpec.describe SAML::SettingsService do
+  let(:success_response_body) { File.read("#{::Rails.root}/spec/fixtures/files/saml_xml/metadata_response_body.txt") }
+  before do
+    # stub out the sleep call to increase rspec speed
+    allow(SAML::SettingsService).to receive(:sleep)
```

Comment:
@aub Stubbing out sleep was very clever and cut the spec run time down from ~3.4 secods to 0.1508 seconds.  Many thanks!